README.md

```diff
@@ -698,7 +704,7 @@ an engine would.
   targets. Implemented in Haxe. [Expat][11].
 * [HaxePunk][179] - A development framework designed to let you build games on
   any platform. [Expat][11].
-* [ImGuizmo][402] - An immediate mode 3D gizmo for scene editing and other controls based on [Dear ImGui][400]. [MIT][403]
+* [ImGuizmo][402] - An immediate mode 3D gizmo for scene editing and other controls based on [Dear ImGui][400]. [/][403]
```

Comment:
It seems you accidentally replaced `MIT` with `/` here. Once this is fixed, this is good to merge :slightly_smiling_face:

Reply:
Oh no.  My mistake.  I believe that I have fixed it in the PR now.